docs/Symbols.md

```diff
@@ -10,4 +10,17 @@ The following are also listed:
 ![An XML document that references two grammars. The referenced grammars are listed in the outline](./images/Symbols/ShowReferencedGrammars.png)
 
 This option has no effect if symbols are disabled through `xml.symbols.enabled`.
-The displayed symbols are affected by `xml.symbols.maxItemsComputed`
\ No newline at end of file
+The displayed symbols are affected by `xml.symbols.maxItemsComputed`
+
+## Max Items Computed
```

Comment:
Please use `xml.symbols.maxItemsComputed` as title to be consiste with Formatting page.